src/rafts/lighthouse/commands/fractal.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable prettier/prettier */
+'use strict';
+/* We did this without ck's help */
+/* eslint-disable-next-line no-unused-vars */
+
+const { createCanvas } = require('canvas');
+
+const Discord = require('discord.js');
+
+const BaseCommand = require('../../BaseCommand');
+
+class FractalCommand extends BaseCommand {
+    constructor(boat) {
+        const options = {
+            name: 'fractal',
+            owner: true,
+            enabled: true,
+        };
+        super(boat, options);
+    }
+
+    run(message) {
+        const width = 1200;
+        const height = 1200;
+        var canvas = createCanvas(width, height);
+        var ctx = canvas.getContext('2d');
+
+        function checkIfBelongsToMandelbrotSet(x, y) {
+            var realComponentOfResult = x;
+            var imaginaryComponentOfResult = y;
+            var maxIterations = 300;
+            for (var i = 0; i < maxIterations; i++) {
+                var tempRealComponent = realComponentOfResult * realComponentOfResult
+                    - imaginaryComponentOfResult * imaginaryComponentOfResult
+                    + x;
+                var tempImaginaryComponent = 2 * realComponentOfResult * imaginaryComponentOfResult
```

Comment:
No vars, change these to be let or const as necessary, prefer const if the values don't change